Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -30,6 +30,8 @@
 #include "config.h"
 #include "CSSPropertyParser.h"
 
+#include "CSSBorderImageSliceValue.h"
+#include "CSSBorderImageWidthValue.h"
```

Comment:
`consumeBorderImageComponents` is in CSSPropertyParserHelpers.cpp already, so you could move `consumeBorderImage` too and then no need to include the headers in both places.

Reply:
Not blocking on this